.tern-project

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is a config for [tern](https://github.com/ternjs/tern) which makes auto suggestion for JS a bit smarter, trying it out.